mne/preprocessing/eyetracking/eyetracking.py

```diff
@@ -164,3 +170,162 @@ def _convert_mm_to_m(array):
 
 def _convert_deg_to_rad(array):
     return array * np.pi / 180.0
+
+
+def convert_units(inst, calibration, to="radians"):
+    """Convert Eyegaze data from pixels to radians of visual angle or vice versa.
+
+    .. warning::
+        Currently, depending on the units (pixels or radians), eyegaze channels may not
+        be reported correctly in visualization functions like :meth:`mne.io.Raw.plot`.
+        They will be shown  correctly in :func:`mne.viz.eyetracking.plot_gaze`.
+        See :gh:`11879` for more information.
+
+    .. Important::
+       There are important considerations to keep in mind when using this function,
+       see the Notes section below.
```

Comment:
@scott-huberty pushed a commit to add this to the top rather than have all the text indented in the `Notes`. It should make `Notes` more readable by itself and also call better attention to it by putting this at the top.